chirp/drivers/tk11.py

```diff
@@ -0,0 +1,3297 @@
+            FrequencyDirection.ADD.label: FrequencyDirection.ADD,
+            FrequencyDirection.REMOVE.label: FrequencyDirection.REMOVE
+        }[value]
+
+
+class Mode(enum.Enum):
+    """Modulation enum"""
+    NFM = 5
+    FM = 0
+    AM = 1
+    LSB = 2
+    USB = 3
+    CW = 4
+
+
+class QTType(enum.Enum):
+    """QT Code type enum"""
+    NONE = 0
+    CTCSS = 1
+    NDCS = 2
+    IDCS = 3
+
+
+class Power(enum.Enum):
+    """Power enum"""
+    LOW = 0
+    MIDDLE = 1
+    HIGH = 2
+
+    @property
+    def level(self):
+        """Get power level"""
+        return {
+            Power.LOW: POWER_LEVELS[0],
+            Power.MIDDLE: POWER_LEVELS[1],
+            Power.HIGH: POWER_LEVELS[2]
+        }[self]
+
+
+class MemoryAddress:
+    """Memory address"""
+    CHANNELS_USAGE = 0x11000
+    FM = 0x12000
+    GENERAL_SETTINGS = 0x13000
+    GENERAL_2_SETTINGS = 0x14000
+    CHANNELS_IDX = 0x15000
+    DTMF_CONTACTS = 0x1A000
+    SCAN_LIST = 0x16000
+    TONE5 = 0x1A800
+    NOAA_DECODE_ADDRESSES = 0x22000
+    NOAA_SAME_EVENT_CONTROL = 0x22280
+
+
+class ChunkSize:
+    """Chunk size"""
+    SETTINGS = 0xF8
+    SETTINGS_2 = 0x70
+    CHANNELS_IDX = 0x10
+    FM = 0x48
+    DTMF_CONTACTS = 0x180
+    SCAN_LIST = 0x80
+    TONE5_CONTACTS = 0x180
+    NOAA_DECODE_ADDRESSES = 0x28
+    NOAA_SAME_EVENTS_CONTROL = 0x80
+
+
+class MessageType(enum.Enum):
+    """Message type enum"""
+    CONNECT_REQUEST = 0x01F4
+    CONNECT_RESPONSE = 0x01F5
+    MEMORY_READ_REQUEST = 0x01FB
+    MEMORY_READ_RESPONSE = 0x01FC
+    MEMORY_WRITE_REQUEST = 0x1FD
+    MEMORY_WRITE_RESPONSE = 0x1FE
+    REBOOT_REQUEST = 0x5DD
+
+
+class ConnectionResponse:
+    """ConnectionResponse"""
+
+    def __init__(self, message_type: int, message_length: int, version: str):
+        self.message_type = message_type
+        self.message_length = message_length
+        self.version = version
+
+    def __str__(self):
+        """Convert representation to string"""
+        return (
+            f"ConnectionResponse | "
+            f"message type : {str(self.message_type)} "
+            f"({self.message_length} bytes) - "
+            f"{self.version}"
+        )
+
+    @staticmethod
+    def from_bytes(bytes_array):
+        """Convert bytes to ConnectionResponse"""
+        if len(bytes_array) < 20:
+            raise errors.InvalidDataError("Connect response invalid format")
+
+        unpacked = struct.unpack('<HH16s', bytes_array[:20])
+
+        return ConnectionResponse(
+            unpacked[0],
+            unpacked[1],
+            unpacked[2].split(b'\x00')[0].decode('ascii', errors='ignore')
+        )
+
+
+class ConnectionRequest:
+    """ConnectionRequest"""
+    MESSAGE_LENGTH = 0x4
+
+    @staticmethod
+    def to_bytes():
+        """Convert request to bytes"""
+        return struct.pack(
+            '<HHI',
+            MessageType.CONNECT_REQUEST.value,
+            ConnectionRequest.MESSAGE_LENGTH,
+            MAGIC_CODE
+        )
```

Comment:
Same here, a class with a single staticmethod might as well be a function. Especially in this case, there aren't even any parameters, so this could be a static bytestring constant.